src/parser/types.ts

```diff
@@ -516,9 +638,31 @@ interface RandomInterruption {
   components: (string)[];
 }
 
+/**  The InterruptionBlock interface is used to define interruptions in a block. These can be used for breaks or attention checks. Interruptions can be deterministic or random. */
 export type InterruptionBlock = DeterministicInterruption | RandomInterruption;
 
-/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened. */
+/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened.
```

Comment:
Consider putting the "If the component is repeated within the block..." section into an info admonition.

Reply:
These are actually wrong. Fixed them up in aa45ce2